SwrveSDK/SwrveEvents.brs

```diff
@@ -354,7 +354,7 @@ function SwrveBuildBatchMeta() as Object
 	jsonObject = {}
 	jsonObject.user = config.userID
 	jsonObject.unique_device_id = config.uniqueDeviceId
-	jsonObject.app_version = config.version
+	jsonObject.app_version = config.appVersion
```

Comment:
Would be useful to know why this was pulling from the version value rather than using the applications version and wether this was intentional or not.

Reply:
Yeah not intentional, should be app version. 'version' is actually the version for the Swrve API for getting campaign and resources.